environment/src/main/resources/schema/app/20200513175538_CB-6764_introduce_serviceEndpointCreation_column_in_environment's_network_entity.sql

```diff
@@ -0,0 +1,9 @@
+-- // CB-6764 introduce serviceEndpointCreation column in environment's network entity
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE environment_network ADD COLUMN IF NOT EXISTS serviceEndpointCreation varchar(255) NOT NULL DEFAULT 'DISABLED';
```

Comment:
you have to add default in separate steps to avoid lock, like this:
https://github.com/hortonworks/cloudbreak/blob/master/freeipa/src/main/resources/schema/app/20200406095012_CB-6385_add_version_to_stack.sql